server/konnectors/currencies.coffee

```diff
@@ -0,0 +1,152 @@
+americano = require 'americano-cozy'
+request = require 'request'
+moment = require 'moment'
+crypto = require 'crypto'
+async = require 'async'
```

Comment:
Here, I would like to import the [xml2js](https://www.npmjs.com/package/xml2js) npm module. Is there a process to add it to Cozy?